config/datasets/coral_test_set.yaml

```diff
@@ -0,0 +1,36 @@
+coral_test_set:
+  speaker_ids:
+    - t16023910
+    - t22996947
+    - t17053799
+    - t47310812
+    - t59821643
+    - t79384144
+    - t37263163
+    - t39126337
+    - t22996947
+    - t17419826
+    - t29982093
+    - t40224720
+    - t27567090
+    - t31776488
+    - t40353825
+    - t34653502
+    - t26322580
+    - t72771561
+    - t48392154
+    - t38841910
+    - t36170006
+    - t10062436
+    - t21820268
+    - t39414039
+    - t13282221
+    - t82923090
+    - t35107011
+    - t13330719
+    - t33840200
+    - t10367179
+    - t39656524
+    - t37619007
+    - t42345465
+    - t21902156
```

Comment:
This doesn't really belong here. The configs in the `datasets` folder should have identical keys, and this config here is about the definition of the test set rather than the usage. Instead, there should be a script which creates the test set and uploads it to HF. Then there can be a config here in `config/datasets`, which specifies the dataset configuration like the other datasets (i.e., `id`, `subset`, and so on). For now, I would move this list to a constant variable in the CoRal dataset build script.